test/JDBC/expected/comment_on_pg_trigger.out

```diff
@@ -0,0 +1,27 @@
+-- psql
+
+CREATE TABLE test1(id1 int);
+GO
+
+CREATE OR REPLACE FUNCTION func1()
+  RETURNS "trigger" AS
+'BEGIN
+    RETURN NEW;
+END;'
+  LANGUAGE 'plpgsql' VOLATILE;
+GO
+
+CREATE TRIGGER trig1 BEFORE UPDATE
+  ON test1 FOR EACH ROW
+  EXECUTE PROCEDURE func1();
+GO
+
+COMMENT ON TRIGGER trig1 ON test1 IS 'hello1';
```

Comment:
what will happen if we try to run COMMENT command on TSQL trigger (created through TDS endpoint)?

Reply:
Comment on TSQL trigger can be added successfully, I've updated the test to check comments on both Postgres and TSQL triggers.